src/sql/platform/accounts/browser/accountManagement.contribution.ts

```diff
@@ -65,12 +65,12 @@ ExtensionsRegistry.registerExtensionPoint<IAccountContrib | IAccountContrib[]>({
 			const iconClass = id;
 			if (typeof icon === 'string') {
 				const path = join(extension.description.extensionLocation.fsPath, icon);
-				createCSSRule(`.icon.${iconClass}`, `background-image: url("${URI.file(path).toString()}")`);
+				createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(URI.file(path))}`);
```

Comment:
This won't work, asCSSUrl expects something with a vscode-remote schema. 

```
import * as resources from 'vs/base/common/resources';
 resources.joinPath(extension.description.extensionLocation, e.path.light),
```

I recommend doing something like this.

Reply:
Well, it'll rewrite the URL if it's remote. But it's a normal URL then it just passes through that part and output the escaped url('...') text. 

If you want to take over the PR to ensure it works for remote I'm fine with that - my goal was just to use the (more thorough) asCssUrl method instead of doing it ourselves.

Reply:
I'm also happy making the changes myself if you can help test. Otherwise the goal here is that this isn't changing existing behavior so if there's bugs with the remote scenario we can also make an item and track that for fixing later.

Reply:
Sure thing, but what I meant was this: https://github.com/microsoft/azuredatastudio/blob/master/src/vs/base/browser/dom.ts#L1190

It'll ignore the URI if the schema isn't vscode-remote (https://github.com/microsoft/azuredatastudio/blob/master/src/vs/base/common/network.ts#L51)

Essentially adding this code would have 0 impact on the uri.

Reply:
I could take over for work on our exploration aspects of the code.